### PR TITLE
Add event log curl examples

### LINKED ISSUE #462: Auditing system events for classic infrastructure: commands in curl/java/go
Could we provide this [Event Logs](https://sldn.softlayer.com/python/event_log/) example in curl/java/go?

I'm working on migrating some SL API information and code examples to the core platform docs for [Auditing system events for classic infrastructure](https://test.cloud.ibm.com/docs/account?topic=account-audit-log) and it would be very helpful to have these core languages represented. 

@allmightyspiff

### PR DESCRIPTION
**Pull request by ATGE**
Fixes #462

### CODE REVIEW

**Review by FernandoOjeda — APPROVED** [review]
Looks good for me.

File: content/rest/events_logs.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+            objectMask=mask[eventName,eventCreateDate,userType]&
+            objectFilter={"eventCreateDate":{"operation":"greaterThanDate","options":[{"name":"date","value":["2021-03-29T00:00:00.0000-06:00"]}]}}
+     
+```
+
+```bash
+curl -u $SL_USER:$SL_APIKEY -X GET -H "Accept: */*" -H "Accept-Encoding: gzip, deflate, compress"  'https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?objectMask=mask%5BeventName%2CeventCreateDate%2CuserType%5D&resultLimit=0%2C50&objectFilter=%7B%22eventCreateDate%22%3A+%7B%22operation%22%3A+%22greaterThanDate%22%2C+%22options%22%3A+%5B%7B%22name%22%3A+%22date%22%2C+%22value%22%3A+%5B%222021-03-01T00%3A00%3A00.0000-06%3A00%22%5D%7D%5D%7D%7D'
+```
+
+Output:
```
Comment (allmightyspiff) [inline content/rest/events_logs.md:58]: Too much output, leave 2 events (first 2 is fine) and remove the rest.

File: content/rest/events_logs.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+```bash
+https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?
+            resultLimit=0,50&
+            objectMask=mask[eventName,eventCreateDate,userType]&
+            objectFilter={"userType":{"operation":"SYSTEM"}}
+
+```
+    
+```bash
+curl -u $SL_USER:$SL_APIKEY -X GET -H "Accept: */*" -H "Accept-Encoding: gzip, deflate, compress"  'https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?objectMask=mask%5BeventName%2CeventCreateDate%2CuserType%5D&resultLimit=0%2C50&objectFilter=%7B%22userType%22%3A+%7B%22operation%22%3A+%22SYSTEM%22%7D%7D'
```
Comment (allmightyspiff) [inline content/rest/events_logs.md:163]:
```suggestion
curl -u $SL_USER:$SL_APIKEY 'https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?objectMask=mask[eventName,eventCreateDate,userType]&resultLimit=0,50&objectFilter={"userType":+{"operation":+"SYSTEM"}}'
```

File: content/rest/events_logs.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+```bash
+https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?
+            resultLimit=0,50&
+            objectMask=mask[eventName,eventCreateDate,userType]&
+            objectFilter={"eventName":{"operation":"^= Login"}}
+```
+
+
+```bash
+curl -u $SL_USER:$SL_APIKEY -X GET -H "Accept: */*" -H "Accept-Encoding: gzip, deflate, compress"  'https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?objectMask=mask%5BeventName%2CeventCreateDate%2CuserType%5D&resultLimit=0%2C50&objectFilter=%7B%22eventName%22%3A+%7B%22operation%22%3A+%22%5E%3D+Login%22%7D%7D'
```
Comment (allmightyspiff) [inline content/rest/events_logs.md:220]:
```suggestion
curl -u $SL_USER:$SL_APIKEY  'https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?objectMask=mask[eventName,eventCreateDate,userType]&resultLimit=0,50&objectFilter={"eventName":+{"operation":+"^=+Login"}}'
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Overall looks great, only complaint is that I would like to have the URLs url-decoded (with something like https://www.urldecoder.org ) so all those http escape characters don't mess-up the readability of the article.

Thanks.

File: content/rest/events_logs.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+```bash
+https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?
+            resultLimit=0,50&
+            objectMask=mask[eventName,eventCreateDate,userType]&
+            objectFilter={"eventCreateDate":{"operation":"greaterThanDate","options":[{"name":"date","value":["2021-03-29T00:00:00.0000-06:00"]}]}}
+     
+```
+
+```bash
+curl -u $SL_USER:$SL_APIKEY -X GET -H "Accept: */*" -H "Accept-Encoding: gzip, deflate, compress"  'https://api.softlayer.com/rest/v3.1/SoftLayer_Event_Log/getAllObjects.json?objectMask=mask%5BeventName%2CeventCreateDate%2CuserType%5D&resultLimit=0%2C50&objectFilter=%7B%22eventCreateDate%22%3A+%7B%22operation%22%3A+%22greaterThanDate%22%2C+%22options%22%3A+%5B%7B%22name%22%3A+%22date%22%2C+%22value%22%3A+%5B%222021-03-01T00%3A00%3A00.0000-06%3A00%22%5D%7D%5D%7D%7D'
```
[the remark this answers is not in the document]
Comment (ATGE) [reply content/rest/events_logs.md:55]: done

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**